lib/file_builder.ex

```diff
@@ -5,57 +5,74 @@ defmodule Floorplan.FileBuilder do
   Takes a list of UrlLink structs and writes to file
```

Comment:
Can we update the docs throughout?  Not just these outdated moduledocs but looks like a lot of the function docs have gotten removed too. It would be nice to have some around the public interfaces.